blockchain/contracts/HelloWorld.sol

```diff
@@ -47,17 +59,29 @@ contract HelloWorld {
         nftOwnership.dropTime = _dropTime;
         nftOwnerships[nftHash] = nftOwnership;
         availableNFTs.push(nftHash);
-        nftsLength = availableNFTs.length;
+        availableNFTsCount = availableNFTs.length;
     }
 
-    function joinDrop() public {
-        // TODO: purchase limit of 5% of all available NFTs
+    function joinDrop(uint256 _numberOfNFTs) public {
         require(
             !joinedUsers.exists(msg.sender),
```

Comment:
is `msg` a given variable name? Just asking because of our guidelines and pronounceable variable names.

Reply:
It's given